app/core/src/main/java/com/fsck/k9/AccountPreferenceSerializer.kt

```diff
@@ -124,7 +125,7 @@ class AccountPreferenceSerializer(
             maxPushFolders = storage.getInt("$accountUuid.maxPushFolders", 10)
             isSubscribedFoldersOnly = storage.getBoolean("$accountUuid.subscribedFoldersOnly", false)
             maximumPolledMessageAge = storage.getInt("$accountUuid.maximumPolledMessageAge", -1)
-            maximumAutoDownloadMessageSize = storage.getInt("$accountUuid.maximumAutoDownloadMessageSize", 32768)
+            maximumAutoDownloadMessageSize = storage.getInt("$accountUuid.maximumAutoDownloadMessageSize", 131072)
```

Comment:
Consider using a constant for the default value of `maximumAutoDownloadMessageSize`. Then there is no need to suppress `MagicNumber`.